src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -923,10 +923,10 @@ public void GetProcesses_EmptyMachineName_ThrowsArgumentException()
 
         [Fact]
         [ActiveIssue(26720, TargetFrameworkMonikers.Uap)]
-        [ActiveIssue(27459)]
         public void GetProcesses_InvalidMachineName_ThrowsInvalidOperationException()
         {
-            Assert.Throws<InvalidOperationException>(() => Process.GetProcesses(Guid.NewGuid().ToString()));
+            Type exceptionType = PlatformDetection.IsWindows ? typeof(InvalidOperationException) : typeof(PlatformNotSupportedException);
+            var exception = Assert.Throws(exceptionType, () => Process.GetProcesses(Guid.NewGuid().ToString()));
```

Comment:
Nit: the `var exception =` isn't needed

Reply:
Thanks. I'm removing this variable in next PR here: https://github.com/dotnet/corefx/pull/27557